Unity/Tests/BananaClient/TcpAdapterTest.cs

```diff
@@ -48,23 +131,46 @@ public void SendingObject()
             "0000 0000 0000 A290"+
             "00");
 
+        byte[] zeros = new byte[520 - rawResponse.Length];
         test_stream.Write(rawResponse, 0, rawResponse.Length);
+        test_stream.Write(zeros, 0, zeros.Length);
 
-        // Wait data transmition. 
-        Thread.Sleep(200);
+        Thread.Sleep(400);
 
-        ConnectServerResponse connectServerResponse = (ConnectServerResponse)tcpAdapter.Receive();
-        Assert.IsNotNull(connectServerResponse);
-        Assert.AreEqual(connectServerResponse.eventType, BytesUtils.HexStringToByteArray("0005"));
-        Assert.AreEqual(connectServerResponse.sessionId, BytesUtils.HexStringToByteArray("0000 0000 0000 0000 0000 0000 0000 0000 00DD 0000"));
+        var response = tcpAdapter.Receive();
+        Assert.IsNotNull(response);
+        var connectServerResponse = (ConnectServerResponse)response;
+        // expect and equal are reversed
+        Assert.AreEqual(connectServerResponse.eventType, 5);
+        // Little Endianess?
```

Comment:
Endianness of primitive types like Int, Long and so on, are up to the OS (Operating System) to decide. I used `BitConverter.IsLittleEndian` to check what endianness of OS. The endianness of byte stream over TCP is Big endian by convention.

Reply:
Umm... May I ask what does "// Little Endianess?" comment mean?

Reply:
The expected value in the UMT should be reversed because the Unit test wrote the data in "Big Endian" directly to the memorystream.